<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserManagerFormUi</class>
 <widget class="SlGfwDialogForm" name="UserManagerFormUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>599</width>
    <height>373</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <widget class="SlGfwLabel" name="InformationLabel">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>280</y>
     <width>393</width>
     <height>51</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="inverseColors">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="SlGfwGroupBox" name="UserPwdGroupBox">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>0</y>
     <width>670</width>
     <height>251</height>
    </rect>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
   <widget class="SlGfwLabel" name="NewPasswordLabel">
    <property name="geometry">
     <rect>
      <x>293</x>
      <y>155</y>
      <width>120</width>
      <height>20</height>
     </rect>
    </property>
    <property name="locale">
     <locale language="Chinese" country="China"/>
    </property>
    <property name="displayMode">
     <enum>SlGfwFunctions::ANY_MODE</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignJustify|Qt::AlignVCenter</set>
    </property>
    <property name="text">
     <string>新密码:</string>
    </property>
   </widget>
   <widget class="SlGfwLineEdit" name="NewPasswordAgainLineEdit">
    <property name="geometry">
     <rect>
      <x>413</x>
      <y>210</y>
      <width>105</width>
      <height>25</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>SlGfwLineEdit::PASSWORD</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading</set>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="inputMode">
     <enum>SlGfwFunctions::ANY_MODE</enum>
    </property>
   </widget>
   <widget class="SlGfwLineEdit" name="UserNameLineEdit">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>168</x>
      <y>155</y>
      <width>102</width>
      <height>25</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>6</verstretch>
     </sizepolicy>
    </property>
    <property name="alignment">
     <set>Qt::AlignHCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="SlGfwLabel" name="OldPasswordLabel">
    <property name="geometry">
     <rect>
      <x>294</x>
      <y>90</y>
      <width>120</width>
      <height>25</height>
     </rect>
    </property>
    <property name="locale">
     <locale language="Chinese" country="China"/>
    </property>
    <property name="displayMode">
     <enum>SlGfwFunctions::ANY_MODE</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignJustify|Qt::AlignVCenter</set>
    </property>
    <property name="text">
     <string>系统现在密码:</string>
    </property>
   </widget>
   <widget class="SlGfwLabel" name="NewPasswordAgainLabel">
    <property name="geometry">
     <rect>
      <x>293</x>
      <y>210</y>
      <width>120</width>
      <height>25</height>
     </rect>
    </property>
    <property name="locale">
     <locale language="Chinese" country="China"/>
    </property>
    <property name="displayMode">
     <enum>SlGfwFunctions::ANY_MODE</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignJustify|Qt::AlignVCenter</set>
    </property>
    <property name="text">
     <string>再次输入新密码:</string>
    </property>
   </widget>
   <widget class="SlGfwLabel" name="label">
    <property name="geometry">
     <rect>
      <x>238</x>
      <y>30</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Siemens AD CH Simplified</family>
      <pointsize>-1</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string>background-color: rgb(255, 255, 0)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="text">
     <string>设定用户密码</string>
    </property>
   </widget>
   <widget class="SlGfwLabel" name="UserNameLabel">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>155</y>
      <width>54</width>
      <height>25</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="inverseColors">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>用户名:</string>
    </property>
   </widget>
   <widget class="SlGfwLineEdit" name="OldPasswordLineEdit">
    <property name="geometry">
     <rect>
      <x>413</x>
      <y>90</y>
      <width>105</width>
      <height>25</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>SlGfwLineEdit::PASSWORD</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading</set>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="inputMode">
     <enum>SlGfwFunctions::ANY_MODE</enum>
    </property>
   </widget>
   <widget class="SlGfwLineEdit" name="NewPasswordLineEdit">
    <property name="geometry">
     <rect>
      <x>413</x>
      <y>150</y>
      <width>105</width>
      <height>25</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>SlGfwLineEdit::PASSWORD</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading</set>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="inputMode">
     <enum>SlGfwFunctions::ANY_MODE</enum>
    </property>
   </widget>
  </widget>
  <widget class="SlGfwGroupBox" name="RepairPwdGroupBox">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>0</y>
     <width>670</width>
     <height>251</height>
    </rect>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
   <widget class="SlGfwLabel" name="NewPasswordLabel_2">
    <property name="geometry">
     <rect>
      <x>293</x>
      <y>155</y>
      <width>120</width>
      <height>20</height>
     </rect>
    </property>
    <property name="locale">
     <locale language="Chinese" country="China"/>
    </property>
    <property name="displayMode">
     <enum>SlGfwFunctions::ANY_MODE</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignJustify|Qt::AlignVCenter</set>
    </property>
    <property name="text">
     <string>新密码:</string>
    </property>
   </widget>
   <widget class="SlGfwLineEdit" name="NewRepairPwdAgainLineEdit">
    <property name="geometry">
     <rect>
      <x>413</x>
      <y>210</y>
      <width>105</width>
      <height>25</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>SlGfwLineEdit::PASSWORD</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading</set>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="inputMode">
     <enum>SlGfwFunctions::ANY_MODE</enum>
    </property>
   </widget>
   <widget class="SlGfwLineEdit" name="RepairNameLineEdit">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>168</x>
      <y>155</y>
      <width>102</width>
      <height>25</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>6</verstretch>
     </sizepolicy>
    </property>
    <property name="alignment">
     <set>Qt::AlignHCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="SlGfwLabel" name="OldPasswordLabel_2">
    <property name="geometry">
     <rect>
      <x>294</x>
      <y>90</y>
      <width>120</width>
      <height>25</height>
     </rect>
    </property>
    <property name="locale">
     <locale language="Chinese" country="China"/>
    </property>
    <property name="displayMode">
     <enum>SlGfwFunctions::ANY_MODE</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignJustify|Qt::AlignVCenter</set>
    </property>
    <property name="text">
     <string>系统现在密码:</string>
    </property>
   </widget>
   <widget class="SlGfwLabel" name="NewPasswordAgainLabel_2">
    <property name="geometry">
     <rect>
      <x>293</x>
      <y>210</y>
      <width>120</width>
      <height>25</height>
     </rect>
    </property>
    <property name="locale">
     <locale language="Chinese" country="China"/>
    </property>
    <property name="displayMode">
     <enum>SlGfwFunctions::ANY_MODE</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignJustify|Qt::AlignVCenter</set>
    </property>
    <property name="text">
     <string>再次输入新密码:</string>
    </property>
   </widget>
   <widget class="SlGfwLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>238</x>
      <y>30</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Siemens AD CH Simplified</family>
      <pointsize>-1</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string>background-color: rgb(255, 255, 0)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="text">
     <string>设定检修密码</string>
    </property>
   </widget>
   <widget class="SlGfwLabel" name="UserNameLabel_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>155</y>
      <width>54</width>
      <height>25</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="inverseColors">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>用户名:</string>
    </property>
   </widget>
   <widget class="SlGfwLineEdit" name="OldRepairPwdLineEdit">
    <property name="geometry">
     <rect>
      <x>413</x>
      <y>90</y>
      <width>105</width>
      <height>25</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>SlGfwLineEdit::PASSWORD</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading</set>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="inputMode">
     <enum>SlGfwFunctions::ANY_MODE</enum>
    </property>
   </widget>
   <widget class="SlGfwLineEdit" name="NewRepairPwdLineEdit">
    <property name="geometry">
     <rect>
      <x>413</x>
      <y>150</y>
      <width>105</width>
      <height>25</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>SlGfwLineEdit::PASSWORD</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading</set>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="inputMode">
     <enum>SlGfwFunctions::ANY_MODE</enum>
    </property>
   </widget>
  </widget>
  <widget class="SlGfwGroupBox" name="DebugPwdGroupBox">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>0</y>
     <width>670</width>
     <height>251</height>
    </rect>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
   <widget class="SlGfwLabel" name="NewPasswordLabel_3">
    <property name="geometry">
     <rect>
      <x>293</x>
      <y>155</y>
      <width>120</width>
      <height>20</height>
     </rect>
    </property>
    <property name="locale">
     <locale language="Chinese" country="China"/>
    </property>
    <property name="displayMode">
     <enum>SlGfwFunctions::ANY_MODE</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignJustify|Qt::AlignVCenter</set>
    </property>
    <property name="text">
     <string>新密码:</string>
    </property>
   </widget>
   <widget class="SlGfwLineEdit" name="NewDebugPwdAgainLineEdit">
    <property name="geometry">
     <rect>
      <x>413</x>
      <y>210</y>
      <width>105</width>
      <height>25</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>SlGfwLineEdit::PASSWORD</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading</set>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="inputMode">
     <enum>SlGfwFunctions::ANY_MODE</enum>
    </property>
   </widget>
   <widget class="SlGfwLineEdit" name="DebugNameLineEdit">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>168</x>
      <y>155</y>
      <width>102</width>
      <height>25</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>6</verstretch>
     </sizepolicy>
    </property>
    <property name="alignment">
     <set>Qt::AlignHCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="SlGfwLabel" name="OldPasswordLabel_3">
    <property name="geometry">
     <rect>
      <x>294</x>
      <y>90</y>
      <width>120</width>
      <height>25</height>
     </rect>
    </property>
    <property name="locale">
     <locale language="Chinese" country="China"/>
    </property>
    <property name="displayMode">
     <enum>SlGfwFunctions::ANY_MODE</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignJustify|Qt::AlignVCenter</set>
    </property>
    <property name="text">
     <string>系统现在密码:</string>
    </property>
   </widget>
   <widget class="SlGfwLabel" name="NewPasswordAgainLabel_3">
    <property name="geometry">
     <rect>
      <x>293</x>
      <y>210</y>
      <width>120</width>
      <height>25</height>
     </rect>
    </property>
    <property name="locale">
     <locale language="Chinese" country="China"/>
    </property>
    <property name="displayMode">
     <enum>SlGfwFunctions::ANY_MODE</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignJustify|Qt::AlignVCenter</set>
    </property>
    <property name="text">
     <string>再次输入新密码:</string>
    </property>
   </widget>
   <widget class="SlGfwLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>238</x>
      <y>30</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Siemens AD CH Simplified</family>
      <pointsize>-1</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string>background-color: rgb(255, 255, 0)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="text">
     <string>设定调试密码</string>
    </property>
   </widget>
   <widget class="SlGfwLabel" name="UserNameLabel_3">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>155</y>
      <width>54</width>
      <height>25</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="inverseColors">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>用户名:</string>
    </property>
   </widget>
   <widget class="SlGfwLineEdit" name="OldDebugPwdLineEdit">
    <property name="geometry">
     <rect>
      <x>413</x>
      <y>90</y>
      <width>105</width>
      <height>25</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>SlGfwLineEdit::PASSWORD</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading</set>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="inputMode">
     <enum>SlGfwFunctions::ANY_MODE</enum>
    </property>
   </widget>
   <widget class="SlGfwLineEdit" name="NewDebugPwdLineEdit">
    <property name="geometry">
     <rect>
      <x>413</x>
      <y>150</y>
      <width>105</width>
      <height>25</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>SlGfwLineEdit::PASSWORD</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading</set>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="inputMode">
     <enum>SlGfwFunctions::ANY_MODE</enum>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SlGfwDialogForm</class>
   <extends>QWidget</extends>
   <header>slgfwdialogform.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SlGfwGroupBox</class>
   <extends>QGroupBox</extends>
   <header>slgfwgroupbox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SlGfwLabel</class>
   <extends>QFrame</extends>
   <header>slgfwlabel.h</header>
  </customwidget>
  <customwidget>
   <class>SlGfwLineEdit</class>
   <extends>QFrame</extends>
   <header>slgfwlineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
